README.md

```diff
@@ -42,9 +39,19 @@ for our CI purposes.
 ```
 Once the above variables are set and the /payload directory has been populated, run the following script:
 ```shell script
-hack/run-ci-e2e-test.sh
+hack/run-ci-e2e-test.sh -k "openshift-dev"
```

Comment:
Add a line mentioning that we are assuming developers are using the `openshift-dev` key in their AWS account.

Reply:
Added a line a below.